alerting/src/main/resources/org/opensearch/alerting/alerts/finding_mapping.json

```diff
@@ -60,6 +60,9 @@
           "type" : "keyword"
         }
       }
+    },
```

Comment:
The schema version for the findings mapping needs to be updated to 3 now.

Reply:
updated